edk2toolext/edk2_invocable.py

```diff
@@ -126,6 +126,7 @@ def ParseCommandLineOptions(self):
             self.PlatformModule = import_module_by_file_name(os.path.abspath(settingsArg.platform_module))
             self.PlatformSettings = locate_class_in_module(
                 self.PlatformModule, self.GetSettingsClass())()
+            self.PlatformSettings.SettingsClass = self.GetSettingsClass()
```

Comment:
This needs to get documented

Reply:
If we already have a method for this, why are we caching it in a variable?

Reply:
So PlatformSettings doesn't have a way to get the current Settings Class. This is giving the settings the class that we looked for. The invocable itself has the method.

Reply:
Documented